spell.ignore

```diff
@@ -0,0 +1,399 @@
+mux
+env
+merchantability
+lmr
+redhat
+rhel
+distro
+rmoura
+libc
+runtime
+stderr
+ctrl
+stty
+paginator
+usr
+conf
+config
+datadir
+dir
+api
+iso
+gpg
+vm
+args
+os
+jeos
+logdir
+rtype
+basestring
+tmp
+cleber
+cli
+cmd
+prepost
+unittests
+ascii
+unicode
+fixme
+html
+json
+sysinfo
+pre
+xunit
+params
+argparse
+urls
+whitespace
+shortname
+url
+str
+traceback
+filename
+namespace
+avocadolib
+docstring
+exc
+tuples
+tuple
+klass
+uid
+areis
+todo
+attr
+muxes
+nics
+sys
+stdout
+splitlines
+throbber
+mem
+func
+fmt
+itertable
+iterable
+ui
+subcommands
+subparsers
+utils
+hostname
+username
+autodetect
+pem
+uptime
+apahim
+proc
+sigterm
+subprocess
+sigstop
+sigtstp
+crosa
+ast
+realname
+bool
+autotest
+py
+intree
+ini
+unittesting
+sigsegv
+booleans
+distros
+logf
+gzipped
+basedir
+profiler
+mbligh
+apw
+shadowen
+mytest
+aaa
+myte
+fs
+dirs
+shlex
+unittest
+sleeptest
+postprocess
+gmail
+yaml
+tty
+ete
+http
+pythonhosted
+phylogenetics
+envs
+throuh
+fileobj
+attrs
+hardcoded
+utf
+uri
+libvirt
+hypervisor
+ip
+https
+githubusercontent
+mcepl
+virt
+addr
+xml
+svn
+jenkins
+ci
+dtkit
+junit
+src
+thalesgroup
+xsd
+testsuite
+timestamp
+testcase
+rogram
+multiline
+motd
+abspath
+multiplexation
+dicts
+restclient
+toplevel
+subcommand
+checksum
+installable
+amd
+libexec
+subparser
+metavar
+signalling
+gdb
+init
+chdir
+ldoktor
+msg
+zipfile
+tarfile
+symlinks
+xz
+rw
+hashfile
+stdlib
+englobing
```

Comment:
This one is located in `avocado/utils/astring.py` and is indeed a valid word, but I can't really understand, what it means. (even from the whole description, I only know I should escape something...)